### PR TITLE
Get it to build

### PR DESCRIPTION
**Pull request by simonvh**
Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

This PR fixes the build for Python 3.6-8.

### CODE REVIEW

**Comment by simonvh** [comment]
@conda-forge-admin, please rerender

**Comment by simonvh** [comment]
@mariusvniekerk can you have a look? It seems to build correctly.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,31 +1,43 @@
...
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+  host:
+    - make
+    - cmake
```
Comment (jakirkham) [inline recipe/meta.yaml]: These can be dropped from `host`. They only need to be in `build`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,31 +1,43 @@
...
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1 
+  skip: True  # [py2k]
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
```
Comment (jakirkham) [inline recipe/meta.yaml]: Is `cmake` needed?

Comment (simonvh) [reply recipe/meta.yaml]: Good call, it's not.

**Comment by simonvh** [comment]
Ok, fixed the dependencies. @jakirkham, if you have time can you take another look?

**Comment by simonvh** [comment]
anyone up for a review :-) @conda-forge/staged-recipes

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,31 +1,40 @@
...
+  host:
     - python
+    - pip
     - setuptools
-    - numpy x.x
+    - numpy 
     - cython
     - scikit-learn
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (isuruf) [inline recipe/meta.yaml]: Remove the 2 lines above. They are only needed in `build`. (Lines 23-24)

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,31 +1,40 @@
...
+{% set hash_val = "dca6cf1bea09cf77cf052119f4a43093" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-learn-contrib/lightning/archive/{{ version }}.tar.gz
```
Comment (isuruf) [inline recipe/meta.yaml:12]: Why this change?

Comment (simonvh) [reply recipe/meta.yaml:12]: The version on pypi does not include the Makefile. Running `make clean` is necessary as the tarball includes some cython-generated files that break the build.

**Comment by jakirkham** [comment]
@conda-forge/sklearn-contrib-lightning, could you please take a look? 🙂

**Comment by simonvh** [comment]
Well, at least there doesn't seem to be any vocal disagreement ;-)

**Comment by simonvh** [comment]
Any updates here?

**Comment by chrisburr** [comment]
@conda-forge/core I guess @conda-forge/sklearn-contrib-lightning aren't available

@simonvh Are you interested in adding yourself as a maintainer?

**Comment by marcelotrevisani** [comment]
@conda-forge-admin, please rerender

**Comment by simonvh** [comment]
@chrisburr sure, I don't mind

**Comment by simonvh** [comment]
Sorry for all you that are tagged here, but I would really like for this PR to be merged. If there's any action I can take I would be happy to do so.

**Comment by scopatz** [comment]
@fabianp @mblondel have not responded in over a week.  Merging
